MorkoUnityProject/Assets/Code/UI/ListItem.cs

```diff
@@ -1,13 +1,13 @@
 ﻿using UnityEngine;
 
-namespace Morko
+
+public class ListItem : MonoBehaviour
 {
-	public class ListItem : MonoBehaviour
-	{
-		public int listItemId;
-		public string listItemName;
+	[SerializeField]private int listItemId;
+	[SerializeField]private string listItemName;
+	[SerializeField] private GameObject model;
 
-		public GameObject[] characters;
-		public GameObject map;
-	}
-}
\ No newline at end of file
+	public int ID { get => listItemId; set => listItemId = value; }
```

Comment:
change to if there is no need to serialize listitemid
public int Id { get; set; }